utils/navigator_tools/navigator_tools/object_database_helper.py

```diff
@@ -0,0 +1,63 @@
+"""Use the DBHelper class to interface with the Database without having to deal with ROS things."""
```

Comment:
It's not meant to be a node, but included in other python files. 

If you would like a node that has this functionality I can write one, but I think it should be a separate file that uses this class.